build.sbt

```diff
@@ -1,12 +1,27 @@
 // Your sbt build file. Guides on how to write one can be found at
 // http://www.scala-sbt.org/0.13/docs/index.html
 
+
+
+//ivyLoggingLevel := UpdateLogging.Full
```

Comment:
please remove or comment on why it should be kept here

Reply:
yeah it's leftover from debugging library dependency, I'll remove it